packages/lab/src/Wizard/WizardActions/WizardActions.tsx

```diff
@@ -26,22 +26,24 @@ export interface HvWizardActionsProps extends HvBaseProps {
   /** An object containing all the labels for the wizard actions component. */
   labels?: {
     /** Cancel button label. */
-    cancel?: string;
+    cancel?: string | React.ReactNode;
     /** Skip button label. */
-    skip?: string;
+    skip?: string | React.ReactNode;
     /** Previous button label. */
-    previous?: string;
+    previous?: string | React.ReactNode;
     /** Next button label. */
-    next?: string;
+    next?: string | React.ReactNode;
     /** Submit button label. */
-    submit?: string;
+    submit?: string | React.ReactNode;
   };
   /** Whether the submit button is disabled. */
   loading?: boolean;
   /** Enables the skip button. */
   skippable?: boolean;
   /** A Jss Object used to override or extend the styles applied to the empty state Wizard. */
   classes?: HvWizardActionsClasses;
+  /** Function executed instead of default go to next page */
+  handleBeforeNext?: () => void;
```

Comment:
Event handlers should be called `on<Event>`, but let's keep the `handle*` for consistency 👍